release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -88,23 +162,50 @@ class CreateReleaseSubstepsState extends State<CreateReleaseSubsteps> {
           setReleaseData: setReleaseData,
           hintText: 'New Dart revision to cherrypick.',
         ),
-        CheckboxListTileDropdown(
+        DropdownAsSubstep(
           index: 7,
           releaseData: releaseData,
           setReleaseData: setReleaseData,
           options: const <String>['y', 'z', 'm', 'n'],
         ),
         const SizedBox(height: 20.0),
-        Center(
-          // TODO(Yugue): Add regex validation for each parameter input
-          // before Continue button is enabled, https://github.com/flutter/flutter/issues/91925.
-          child: ElevatedButton(
-            key: const Key('step1continue'),
-            onPressed: () {
-              widget.nextStep();
-            },
-            child: const Text('Continue'),
+        if (_error != null)
+          Center(
+            child: SelectableText(
+              presentError(_error),
+              style: Theme.of(context).textTheme.subtitle1!.copyWith(color: Colors.red),
+            ),
           ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            ElevatedButton(
+              key: const Key('step1continue'),
+              onPressed: _isLoading
+                  ? null // if the release initialization is loading, disable this button
+                  : () async {
+                      setError(null);
```

Comment:
why do we clear out the error here?

Reply:
yea, this makes sure there is no error present during the `loading` state. For example, a user uses the conductor, and gets an error. After, the user corrects the inputs and hits `continue` again. `continue` becomes disabled, and a loading animation appears. Without setting the error object to null, the user would still see the error they got previously during the loading phase.

Reply:
ahh, i see. that makes sense.

Reply:
as a nit, maybe in the `setError()` method add a check that the value we're trying to set this to isn't already the value of `_error`. if it is, then no need to call `setState` and no need to re-rebuild the widget tree.